app/imports/collections/applicationSections.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from 'meteor/meteor';
+
+class ApplicationSectionsCollection extends Mongo.Collection {
+  insert(doc, callback) {
+    doc.createdAt = doc.createdAt || new Date();
+    doc.updatedAt = doc.updatedAt || new Date();
+    doc.userId = Meteor.userId();
+    doc.year = doc.year || new Date().getFullYear();
```

Comment:
is there point in this as createdAt will have this year value. or will we most likely set this manually?